services/GoogleAds.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Google Ads",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://payments.google.com/payments/apis-secure/get_legal_document?ldi=31472",
+      "select": "body"
+    },
+    "Parent Organization Privacy Policy": {
+      "fetch": "https://policies.google.com/privacy?hl=en-GB",
+      "filter": [
+        "removeUTMfromUrls"
+      ],
+      "select": "div[role=article]",
+      "remove": ".KMMDve, .TL2alc, img[src$='.svg']"
+    },
+    "Program Policies": {
+      "fetch": "https://support.google.com/adspolicy/answer/6008942",
+      "select": ".article-container"
+    },
+    "Data Controller Agreement": {
+      "fetch": "https://privacy.google.com/businesses/controllerterms/",
+      "select": ".ads-controller"
+    },
+    "Data Processor Agreement": {
+      "fetch": "https://privacy.google.com/businesses/processorterms/",
+      "select": ".ads-controller"
+    },
+    "Cookies Policy": {
```

Comment:
Since this morning `Cookies Policy` is renamed to `Trackers Policy`

Reply:
Yes :) I've just refactored a few seconds ago !